dstat.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: dstat
+  version: 0.7.4
+  epoch: 0
+  description: A versatile resource statistics tool
+  copyright:
+    - license: GPL
```

Comment:
Is this right? Should it be `GPL-2.0-only`?

Reply:
it should be yes, thank you.